src/integ_test/java/games/strategy/engine/lobby/server/db/BadWordControllerTest.java

```diff
@@ -0,0 +1,21 @@
+package games.strategy.engine.lobby.server.db;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import games.strategy.util.Util;
+
+public class BadWordControllerTest {
```

Comment:
Sounds good.  I've only found the difference in naming necessary if some build operation needs to be applied to integration tests vs. unit tests, but I think we have that covered with the separate Gradle source set.  In the future, if we need some way to distinguish these types of tests at runtime, it might be worth considering an annotation-based approach instead.